### PR TITLE
Display day of week in event header date

### PR DESCRIPTION
**Pull request by DavidAndreev**

### CODE REVIEW

**Comment by nop33** [comment]
Can you also please add a screenshot so we don't have to check it out manually? Cheers

**Comment by DavidAndreev** [comment]
header

**Comment by DavidAndreev** [comment]
Or should I add coma after the day of week ?

**Comment by DavidAndreev** [comment]
It looks the same, after the arrow it says the day of the week and then the date and time, just like starting date, sorry but I already rebased to devel so I cant take screenshot

File: indico/modules/events/templates/display/indico/_common.html [hunk 1]
```diff
@@ -89,6 +89,7 @@
 
 {% macro render_datetime(prop_name, dt, timezone, format='medium') -%}
     <time itemprop="{{ prop_name }}" datetime="{{ dt.isoformat() }}">
+        {{ dt.strftime('%A') }}
```
Comment (ThiefMaster) [inline indico/modules/events/templates/display/indico/_common.html]: I'd rather `| format_date(...)` this so we keep the correct timezone and locale. Also make sure to test it with another language setting to make sure you get the weekday in that language.

Comment (ThiefMaster) [inline indico/modules/events/templates/display/indico/_common.html]: trans is used to translate static text; but if you use `format_date` it'll properly translate weekdays etc

Comment (DavidAndreev) [inline indico/modules/events/templates/display/indico/_common.html]: dates
Im not sure what you mean, i get this when using format='full' , its good but that timezone name is really long

Comment (ThiefMaster) [inline indico/modules/events/templates/display/indico/_common.html]: You can put custom format strings there: http://babel.pocoo.org/en/latest/dates.html#date-fields

Comment (DavidAndreev) [inline indico/modules/events/templates/display/indico/_common.html]: Yeah that works and also gets translated, thanks

**Comment by ThiefMaster** [comment]
Do we want to show this for multi-day events at all?

**Comment by pferreir** [comment]
No, this is not needed for multi-day events. Those should have the weekday visible in the header of each day.

File: indico/modules/events/templates/display/indico/_common.html [hunk 2]
```diff
@@ -87,8 +87,11 @@
     {% endfor %}
 {%- endmacro %}
 
-{% macro render_datetime(prop_name, dt, timezone, format='medium') -%}
+{% macro render_datetime(prop_name, dt, timezone, format='medium', show_week_day=false) -%}
```
Comment (ThiefMaster) [inline indico/modules/events/templates/display/indico/_common.html]: `show_weekday`

**Pull request merged**